src/index.js

```diff
@@ -0,0 +1,59 @@
+function myFunction() {
+  let itm = document.querySelector("#myList1 > div");
+  let cln = itm.cloneNode(true);
+  document.getElementById("myList1").appendChild(cln);
+}
+let button = document.getElementById("button");
+button.addEventListener("click", myFunction);
+
+let button2 = document.getElementById("button2");
+button2.addEventListener("click", myFunctionTwo);
+
+function myFunctionTwo() {
+  let itm = document.querySelector("#myList2 > div");
+  let cln = itm.cloneNode(true);
+  document.getElementById("myList2").appendChild(cln);
+}
+
+// THE CODE ABOVE, CLONES THE INPUTS^
+let addResult = document.getElementById("add-result");
+addResult.addEventListener("click", startBalanse);
+
+function startBalanse() {
```

Comment:
I propose to use calculateBalance or sumBalance for the function. start is appropriate verb for some process